### ISSUE TITLE
Web UI

### ISSUE THREAD

**Issue by ronwarner** [unixba/b64pack#2]
Have you made, or do you know of any web sites that use this library on the back-end, and provide and input and output box to use it?

**Comment by unixba** [comment]
No. B64pack is a standalone utility meant to be run from the command
line. It served as a proof of concept for a paper and has not been
maintained since.

-- 
Kenan
